cpp/include/cudf/strings/detail/utilities.cuh

```diff
@@ -139,11 +140,14 @@ auto make_strings_children(SizeAndExecuteFunction size_and_exec_fn,
 
   // Compute the offsets values
   for_each_fn(size_and_exec_fn);
-  thrust::exclusive_scan(
-    rmm::exec_policy(stream), d_offsets, d_offsets + strings_count + 1, d_offsets);
+
+  auto const bytes = cudf::detail::exclusive_scan_reduce(
+    d_offsets, d_offsets + strings_count + 1, d_offsets, stream);
+  CUDF_EXPECTS(bytes <= static_cast<size_t>(std::numeric_limits<size_type>::max()),
+               "Size of output exceeds column size limit");
 
   // Now build the chars column
-  auto const bytes = cudf::detail::get_value<int32_t>(offsets_view, strings_count, stream);
+  // auto const bytes = cudf::detail::get_value<int32_t>(offsets_view, strings_count, stream);
```

Comment:
nit: commented out code.

Reply:
Yes, thanks. This is not ready for review.